composer.json

```diff
@@ -22,11 +22,13 @@
         }
     },
     "require": {
-        "php": "^7.1"
+        "php": ">=7.1"
```

Comment:
This version is too loosely constrained. Can you make it "^7.2|^8.0" to make sure php 9.0 throws an error.